test/e2e/e2e_test.go

```diff
@@ -187,9 +130,112 @@ var _ = Describe("e2e", func() {
 			// Check that planner-agent service is running
 			Eventually(func() bool {
 				return agent.IsServiceRunning(agentIP, "planner-agent")
-			}, "3m").Should(BeTrue())
+			}, "6m").Should(BeTrue())
 
 			WaitForAgentToBeUpToDate(source.Id)
 		})
 	})
 })
+
+var _ = Describe("e2e-download-ova-from-url", func() {
+
+	BeforeEach(func() {
+		if testOptions.downloadImageByUrl == false {
```

Comment:
Can you please set here just 
`testOptions.downloadImageByUrl = true`
ANd in the test cases above:
`testOptions.downloadImageByUrl = false`